### PR TITLE
chore: add volunteer nouraellm

### PR DESCRIPTION
**Pull request by nouraellm**

### CODE REVIEW

File: data/users/nouraellm.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Noura EL ALLAM",
+  "bio": "Programmer",
+  "image": "https://avatars.githubusercontent.com/u/26191219?v=4",
+  "categories": [
+    {
+      "slug": "programming",
+      "topics": ["Python", "C", "Data structures", "Algorithms", "Odoo"]
```
Comment (RakeshSangem) [inline data/users/nouraellm.json:8]: Limit these topics to 3.

**Comment by codewithvoid** [comment]
Thanks for volunteering @nouraellm . Can you please make the changes @RakeshSangem suggested

**Comment by nouraellm** [comment]
@codewithvoid @RakeshSangem thanks for the review! changes ✅ .

**Comment by codewithvoid** [comment]
Thanks you @nouraellm for making the changes. Welcome!

**Pull request merged**